pkg/config/loader/config.go

```diff
@@ -28,6 +32,8 @@ func LoadYamlConfig(configObject interface{}, configFilePaths ...string) (*YAMLM
 
 	for _, filePath := range configFilePaths {
 		if fileExists(filePath) {
+			absPath, _ := filepath.Abs(filePath)
```

Comment:
why ignore error?

Reply:
I didn't want to change behaviour to add this log. I don't expect fileExists to work and Abs not to do it, but just in case I wanted to maintain behaviour. (Maybe I should add a comment?)

Reply:
Fair enough